addons/web/static/tests/fields/relational_fields/field_many2one_tests.js

```diff
@@ -2926,6 +2926,60 @@ QUnit.module('fields', {}, function () {
             form.destroy();
         });
 
+        QUnit.test('search more in many2one: dropdown click', async function (assert) {
+            assert.expect(8);
+
+            for (let i = 0; i < 8; i++) {
+                this.data.partner.records.push({id: 100 + i, display_name: 'test_' + i});
+            }
+
+            // simulate css using matching modal inside the dom (not at first child)
```

Comment:
maybe say that the field html renders an element that looks like a modal but which isn't, and that's what you try to
reproduce here

Reply:
> // simulate modal-like element rendered by the field html